In Void cross-build environments, $CC and $CXX may actually contain arguments.

--- a/setup.py	2024-11-26 09:12:22.059041830 -0500
+++ b/setup.py	2024-11-26 09:12:44.641146370 -0500
@@ -92,7 +92,7 @@
     )
     cxx = os.environ.get("CXX", "c++")
     cpp_test = subprocess.Popen(
-        [cxx, "-x", "c++", "-std=c++14", "-"],
+        shlex.split(cxx) + ["-x", "c++", "-std=c++14", "-"],
         stdin=PIPE,
         stdout=PIPE,
         stderr=PIPE,
